### PR TITLE
#54 Fixed flow of booking tickets from the device keyboard 

### PR DESCRIPTION
**Pull request by mathvaillant**

### CODE REVIEW

File: src/screens/TicketSelection/TicketSelection.tsx [hunk 1]
```diff
@@ -1,93 +1,143 @@
...
+    replace(NavigationScreens.BOOKING_SUMMARY);
   };
 
   const onPressBook = () => {
-    if (!hasTicket()) return;
-    replace(NavigationScreens.PAYMENT, {
-      previousScreen: NavigationScreens.TICKET_SELECTION,
-      ticketGroup: ticketGroupName,
-    });
+    if (!hasTicketSelected.value) return;
```
Comment (mathvaillant) [inline src/screens/TicketSelection/TicketSelection.tsx:92]: Should we have some kind of feedback so they know they did not select any tickets?

File: src/screens/BookingSummary.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```
Comment (mathvaillant) [inline src/screens/BookingSummary.tsx:1]: Reason: The hooks being used inside this component are complaining about the `computed` value, which does not need to be included in the deps array.

File: src/screens/Payment.tsx [hunk 3]
```diff
@@ -1,12 +1,14 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```
Comment (mathvaillant) [inline src/screens/Payment.tsx:1]: Reason: The hooks being used inside this component are complaining about the `computed` value, which does not need to be included in the deps array.

File: android/app/src/main/java/com/flexwaysfrs/MainActivity.java [hunk 4]
```diff
@@ -72,21 +79,22 @@ public boolean dispatchKeyEvent(KeyEvent event) {
...
 
     if (
-      keyCode == KeyEvent.KEYCODE_ENTER || 
-      keyCode == KeyEvent.KEYCODE_SPACE || 
+      keyCode == KeyEvent.KEYCODE_ENTER ||
+      keyCode == KeyEvent.KEYCODE_SPACE ||
       keyCode == KEYCODE_FN
     ) {
       if (event.getAction() == KeyEvent.ACTION_DOWN) {
+        Log.d("MainActivity", "dispatchKeyEvent: " + event.getKeyCode());
```
Comment (mathvaillant) [inline android/app/src/main/java/com/flexwaysfrs/MainActivity.java:87]: Just to be faster next time we need to check the logcat on Android Studio

File: src/screens/TicketSelection/TicketSelection.tsx (hunk 1 above)
Comment (joel-gouveia) [reply src/screens/TicketSelection/TicketSelection.tsx:92]: Yes, flag that to Sara but lets add a ticket to add that later

**Review by joel-gouveia — APPROVED** [review]
LGTM

**Pull request merged**